datastore/tests/unit/test_client.py

```diff
@@ -860,13 +860,44 @@ def test_allocate_ids_w_partial_key(self):
         # Check the IDs returned.
         self.assertEqual([key._id for key in result], list(range(num_ids)))
 
-    def test_allocate_ids_with_completed_key(self):
+    def test_allocate_ids_w_completed_key(self):
```

Comment:
I renamed a function to match the naming  conventions with the other functions which is already there.